test_holiday_functions/test_orange_holiday_functions.py

```diff
@@ -17,7 +17,7 @@
 
 
 # Asante
-
+assert not home_for_the_holidays("Maqubela-Flinstone") #name is longer than 13 characters long
```

Comment:
Most Python formatters will add a space after the comment symbol
 
```suggestion
assert not home_for_the_holidays("Maqubela-Flinstone") # name is longer than 13 characters long
```